playbooks/roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
If the customer is running a deployment in a docker container does this require us to rebuild the images?